File: hello.cpp
#include<iostream>    // attached a header file


int main()
{
	
	/* initialized variables */
	
	unsigned short int roll_num = 0;
	char name[32] = 'xxxxxx';
	
	/* taking user input */
	
	std::cout << "Hello World" << std::endl;
	std::cout << "Enter name :";
	std::cin.getline(name);
	std::cout << "Enter roll number :" << std::endl;
	std::cin >> roll_num;
	
	/* printing user input on standart output (console window) */
	
	std::cout << name << "your roll number is " << roll_num;
	return 0;
}

File: hello.py
from sys import stdout as outfile    # added stdout as outfile from sys module


roll_num, name = int(input("Enter roll number :"), input("Enter name :")          # took user input for roll number and name
print("Hello World," + name + "your roll number is" + str(roll_num), sep = ' ', end = '\n', flush = True, file = outfile)
                  # multiline comment
''' print name and roll number on standard output without loading it into buffer '''
